src/config/network/wpa_supplicant.md

```diff
@@ -3,7 +3,29 @@
 The `wpa_supplicant` package is installed by default on the base system. It
 includes utilities to configure wireless interfaces and handle wireless security
 protocols. To use wpa_supplicant, you will need to enable [the wpa_supplicant
-service](#the-wpa_supplicant-service).
+service](../services/index.md#enabling-services). 
```

Comment:
https://docs.voidlinux.org/config/network/wpa_supplicant.html#the-wpa_supplicant-service has a link to the part about enabling services...